src/renderer/features/governance/lib/unlock.ts

```diff
@@ -0,0 +1,24 @@
+import { Address } from '@shared/core';
+import { Chunks, UnlockChunk } from '@shared/api/governance';
+
+export const unlockService = {
+  getSecondsDuratonToBlock,
+  filterClaims,
+};
+
+function getSecondsDuratonToBlock(timeToBlock: number): number {
+  const currentTime = new Date().getTime();
```

Comment:
What do you think to use `Date.now()` here? It's a little bit faster and more meaningful